### PR TITLE
Bug 568/announcements not able to be created

### PR DESCRIPTION
**Pull request by billyeh**
Need to use `ModelMultipleChoiceField` instead of `ModelChoiceField` and add `save_m2m` for the created object.

### CODE REVIEW

File: ap/announcements/views.py [hunk 1]
```diff
@@ -34,6 +34,7 @@ def form_valid(self, form):
     req = form.save(commit=False)
     req.trainee_author = trainee_from_user(self.request.user)
     req.save()
+    form.save_m2m()
```
Comment (jontey) [inline ap/announcements/views.py]: +1

**Comment by ksungreborn** [comment]
I notice announcements created by trainees have no way to view it's status and no way of being approved as of now.

**Comment by jontey** [comment]
Is view announcements only for TAs?

**Comment by billyeh** [comment]
There's the view announcements page which is supposed to show which announcements they're supposed to announce in class that day. It should be a printable report, I think, that only TAs can view?

**Comment by jontey** [comment]
Ok. Then the link on the smart menu should be changed to TA only

**Comment by billyeh** [comment]
It is. https://github.com/attendanceproject/djattendance/blob/40cd49e2febf36ba90a0e726ed90f29c093c25d6/ap/aputils/templatetags/smart_menu.py#L100 

But one thing we can do is move the announcement-request-list link to common, rather than being in both trainee and TA.

**Comment by jontey** [comment]
Didn't do a pull before I started reviewing :S

**Comment by ksungreborn** [comment]
@billyeh I wanted to put announcement-request-list in common, but I felt the name for the tab for TA should be 'Create/Approve Announcements' and not just 'Create Announcements', because TA's will be approving not just creating.

**Comment by billyeh** [comment]
Good point!

**Pull request merged**